src/file-uploader.scss

```diff
@@ -38,19 +37,12 @@ $block: #{$fd-namespace}-file-uploader;
 
     @include fd-focus() {
       outline: none;
-
-      ~ .#{$block}__button {
```

Comment:
@manu-kr what was the reason for that?

Reply:
@droshev I have included button.scss because of i was needed below mixins i.e defined in button.scss.

@mixin buttonContainerFocus() {
  box-shadow: none;

  @include standardFocus();

  &.#{$block}--ghost {
    @include ghostFocus();
  }

  &.#{$block}--positive {
    @include positiveFocus();
  }

  &.#{$block}--negative {
    @include negativeFocus();
  }

  &.#{$block}--attention {
    @include attentionFocus();
  }

  &.#{$block}--emphasized {
    @include negativeFocus();
  }
}



@mixin buttonContainerHover() {
  @include standardHover();

  &.#{$block}--ghost {
    @include ghostHover();
  }

  &.#{$block}--positive {
    @include positiveHover();
  }

  &.#{$block}--negative {
    @include negativeHover();
  }

  &.#{$block}--attention {
    @include attentionHover();
  }

  &.#{$block}--emphasized {
    @include emphasizedHover();
  }
}

These mixins are already defined in button.scss so i have included this just for reusing.

Reply:
@manu-kr why would you need them? for the button?

Reply:
I think it would be for validation before uploading a file
@manu-kr can you confirm